app/models/plan.rb

```diff
@@ -573,6 +573,14 @@ def owner
   end
 
   ##
+  # the owner and co-owners of the project
+  #
+  # @return [Users]
+  def owner_and_coowners
+    return User.joins(:roles).where("roles.plan_id = ? AND roles.access >= ?", self.id, 14)
```

Comment:
Nice to see queries to db in the right place. Remember ruby returns the last value executed implicitly so not really needed the return in this case, but it is just a personal preference.